exercises/dominoes/description.md

```diff
@@ -5,9 +5,9 @@ correct domino chain (the dots on one half of a stone match the dots on the
 neighbouring half of an adjacent stone) and that dots on the halfs of the stones
 which don't have a neighbour (the first and last stone) match each other.
 
-For example given the stones `21`, `23` and `13` you should compute something
-like `12 23 31` or `32 21 13` or `13 32 21` etc, where the first and last numbers are the same.
+For example given the stones `(2, 1)`, `(2, 3)` and `(1, 3)` you should compute something
```

Comment:
Would square brackets be more appropriate here?
And is "comma space" the best separator?

Possible alternatives:
`[1, 1]` `[2,2]` `[3|3]` `[4:4]`